packages/usql/build.sh

```diff
@@ -2,20 +2,42 @@ TERMUX_PKG_HOMEPAGE=https://github.com/xo/usql
 TERMUX_PKG_DESCRIPTION="A universal command-line interface for SQL databases"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@flipee"
-TERMUX_PKG_VERSION="0.19.3"
+TERMUX_PKG_VERSION="0.19.4"
 TERMUX_PKG_SRCURL=https://github.com/xo/usql/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=3955e13c7108cb4a3ebb968dafa501db72d4f4fa0ddf6fb128cceb84e5a3d4c4
+TERMUX_PKG_SHA256=61aff0f9448ca9d90894bdf99276186f87b5a91ab196ff45133f34c61475b717
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
+termux_step_post_get_source() {
+	termux_setup_golang
+	go mod tidy
+	go mod vendor
+}
+
 termux_step_make() {
 	termux_setup_golang
-	export GOPATH=$TERMUX_PKG_BUILDDIR
 
-	cd $TERMUX_PKG_SRCDIR
+	# Build and replace resvg
+	local _resvg_go_url="$(cat go.mod | grep resvg | awk '{print $1}')"
```

Comment:
Bash does have built-in syntax for grabbing the contents of a file in the form of `$(</path/to/file)`

Quote:[^1]
```bash
The command substitution $(cat file) can be replaced by the
equivalent but faster $(< file).
```
Though I don't think this would make any practical difference in this instance or the one below.

[^1]: https://www.man7.org/linux/man-pages/man1/bash.1.html#EXPANSION:~:text=%24(%3C